Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -129,6 +129,9 @@ public struct RenderNodeTranslator: SemanticVisitor {
         node.hierarchy = hierarchy.hierarchy
         node.metadata.category = technology.name
         
+        let documentationNode = try! context.entity(with: identifier)
```

Comment:
Definitely out-of-scope for this PR but it seems like the `visitTutorial` method should `throw` at this point.

Reply:
In this case, it's actually a programmer error if `context.entity(with:)` throws, so the force-unwrap makes sense. It could be improved with a `preconditionFailure` with a more user-facing message, though.